src/modules/Dashboard/features/Applet/AddUser/AddUser.tsx

```diff
@@ -18,7 +17,7 @@ export const AddUser = () => {
 
   useBreadcrumbs([
     {
-      icon: <Svg id="users-outlined" width="15" height="15" />,
+      icon: 'users-outlined',
```

Comment:
here width and height are different: 15, not 18
is the icon properly shown in breadcrumbs?

Reply:
yes, it is. checked all cases, all icons are disaplyed correctly